hooks/post-command

```diff
@@ -72,14 +72,22 @@ get_codecov_uploader() {
   fi
 
   # One-time step
-  curl \
-      -fSs \
-      --retry 5 \
-      https://keybase.io/codecovsecurity/pgp_keys.asc \
+  if [[ "${BUILDKITE_PLUGIN_CODECOV_PGP_PUBLIC_KEY_URL:-false}" = false ]]; then
+    cat pgp_keys.asc \
```

Comment:
Can we add a path here, do it doesn't matter from where this file is executed, please?

Reply:
Addressed here: https://github.com/joscha/codecov-buildkite-plugin/pull/28/commits/f69e53c6009e1aedb346796f63387e65559fa35c

Reply:
I added `$(pwd)`. Not sure if that cuts it, please let me know if you have a better suggestion though.